### DISCUSSION TITLE
iTwin.js loaded map

### DISCUSSION THREAD

**Discussion by myl-Giser** [iTwin/itwinjs-core#6281] [Q&A]
When I loaded the map using iTwin.js, the program appeared on some computers that the earth was loaded as a cylinder instead of a sphere，but some computers  are normal spheres.why?Do you need to configure any parameters?
The first picture is error result，the second one is correct result.
![error](https://github.com/iTwin/itwinjs-core/assets/81339378/2603ea77-0487-4bc0-8fc2-796e26724d02)
![image](https://github.com/iTwin/itwinjs-core/assets/81339378/e5807d6a-698f-4d1b-8af1-cfa5f89ba379)

**Comment by aruniverse** [comment]
What version of itwin.js are you using?
What browsers are affected?
Any differences between the machines where it works vs doesnt?
Is this happening in a specific imodel?
Can you share any code in a public repo or sandbox?

**Reply by myl-Giser** [reply]
What version of itwin.js are you using?
The version of itwin.js is 3.4.3. 
What browsers are affected?
It's the same in any browsers.
Any differences between the machines where it works vs doesnt?
There are no special differences between the machines.
Is this happening in a specific imodel?
Yes，some models can load normally.
Can you share any code in a public repo or sandbox?
I'm so sorry this code is too big, I can provide the code to load the map.

 const vpDiv = IModelApp.viewManager.selectedView;
        const settings = BaseMapLayerSettings.fromJSON({
          formatId: "TileURL",
          url:"https://server.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer/tile/{level}/{row}/{column}/",
          name: "arcgisonline",
        });
        vpDiv!.view.displayStyle.backgroundMapBase = settings;
Thanks for you feedback.

**Reply by aruniverse** [reply]
Please update to the latest 3.x version and see if the issue still persists
